{"name": "test_b3", "status": "failed", "statusDetails": {"message": "AssertionError: Incorrect\nassert 3 == 4", "trace": "def test_b3():\n        a = 3\n        b = 4\n>       assert a == b, \"Incorrect\"\nE       AssertionError: Incorrect\nE       assert 3 == 4\n\nInheri_test.py:28: AssertionError"}, "start": 1650035023300, "stop": 1650035023300, "uuid": "72ea16d9-d153-48a2-a4fa-8673f9caea87", "historyId": "551064e963d0ad2e63d68ef9e0560c24", "testCaseId": "8451ecd051e839f5a6e4f5936e89878f", "fullName": "Inheri_test#test_b3", "labels": [{"name": "suite", "value": "Inheri_test"}, {"name": "host", "value": "MSDT120"}, {"name": "thread", "value": "19292-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Inheri_test"}]}